### ISSUE TITLE
The mysterious CEGUI artefact

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#752]
Quoting myself from #746:

> I think I spotted one CEGUI issue, but I haven't found how to reproduce it yet. I had the game playing in background while reviewing the commits here, and when I came back to the game there was a very small circle displayed on the screen (like 4px of diameter or something), and it would move a bit when I approached it with the cursor. My guess would be that it was an empty tooltip (with just the four corners of the tooltip box, making it look like a small circle). I haven't found yet how to reproduce it though, and it shouldn't prevent merging.

I reproduced it today when letting the game idle in the background, so here is a screenshot:
![od1](https://cloud.githubusercontent.com/assets/4701338/7650898/409b0902-fafb-11e4-8561-96a2699457fa.jpg)

**Comment by Bertram25** [comment]
This looks definitely a CEGUI empty tooltip.
AFAIK, as mouse move events are injected by us, CEGUI might have edge cases where the tooltip is not well handled.

Maybe there is a way to disable the tooltip of the main DefaultWindow of those modes. I'll have a look. (But can't promise an easy win, yet.)

**Comment by hwoarangmy** [comment]
BTW, I confirm it is the same thing as I have.

**Comment by Bertram25** [comment]
Moving to 0.7 as I need time to investigate and it's no blocker.

**Comment by akien-mga** [comment]
> This looks definitely a CEGUI empty tooltip.

What about disabling tooltips completely on focus loss? As far as I know this artefact only shows up when playing in windowed mode and after having done something else than playing OD (maybe it's the dungeon keeper punishing non-focused players :P).

**Comment by Bertram25** [comment]
> What about disabling tooltips completely on focus loss? As far as I know this artefact only shows up when playing in windowed mode and after having done something else than playing OD (maybe it's the dungeon keeper punishing non-focused players :P).

That's an idea. :) I'll check both OD and Cegui code about this.

**Comment by Bertram25** [comment]
@hwoarangmy Once we have a first usable cegui 0.8.5 snapshot, I'll have a look at whether this bug is gone. :)

**Comment by hwoarangmy** [comment]
Yep. Let's hope it will be gone ^^

**Comment by Bertram25** [comment]
0.8.7 is out! :D Time to talk to @muammar ;)

**Comment by muammar** [comment]
@Bertram25 yes, I have seen. One of the CEGUI's upstream wrote about this. I hope I upload it this weekend.

**Comment by Bertram25** [comment]
@muammar Thanks in advance! Feel free to tell me so that I'll hint the cegui team in the corresponding thread.

**Comment by muammar** [comment]
@Bertram25 the package is in [NEW](https://ftp-master.debian.org/new/cegui-mk2_0.8.7-1.html). I hope this week is accepted.

**Comment by muammar** [comment]
@Bertram25 the package has just been accepted in the Debian archives.

**Comment by Bertram25** [comment]
@muammar Thanks! :D I'll check it out asap!
